packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -340,10 +342,11 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                     ...this.hostPolicy.snapshotOptions,
                 };
 
-                // No limit on size of snapshot, as otherwise we fail all clients to summarize
-                if (this.hostPolicy.summarizerClient) {
-                    snapshotOptions.mds = undefined;
-                }
+                const maxSnapshotSizeLimit = 25000000; // 25 MB
+                const maxSnapshotFetchTimeout = 30000; // 30 sec
```

Comment:
move these inline above

Reply:
const snapshotOptions: ISnapshotOptions = {
                    deltas: 1,
                    channels: 1,
                    blobs: 2,
                    ...this.hostPolicy.snapshotOptions,
                   mds: snapshotOptions.mds ? Math.min(snapshotOptions.mds, maxSnapshotSizeLimit) : maxSnapshotSizeLimit,
                   timeout:  snapshotOptions.timeout ? Math.min(snapshotOptions.timeout, maxSnapshotFetchTimeout) : maxSnapshotFetchTimeout
                };

---
In reply to: [509818589](https://github.com/microsoft/FluidFramework/pull/4026#discussion_r509818589) [](ancestors = 509818589)

Reply:
done.